conf/messages

```diff
@@ -181,7 +181,7 @@ hts.helpers.sidebar.government-gateway=Government Gateway
 hts.helpers.sidebar.universal-credit=Universal Credit
 hts.helpers.sidebar.apply-for-universal-credit=Apply for Universal Credit
 hts.helpers.sidebar.working-tax-credit=Working Tax Credit
-hts.helpers.phaseBanner.feedback=This is a new service – your feedback will help us to improve it.
+hts.helpers.phaseBanner.feedback=This is a new service – your <a href="{0}">feedback</a> will help us to improve it.
```

Comment:
Do we want HTML in our messages file? It hides it from the view file where the reset of the HTML is

Reply:
We don't need that anyway, so I have removed that along with the phaseBanner helper.